robot_skills/src/robot_skills/arms.py

```diff
@@ -318,6 +322,188 @@ class GripperState(object):
     OPEN = "open"
     CLOSE = "close"
 
+class ArmPiece(object):
```

Comment:
Still doubting the name 'Piece'. I'd expect that it will all be 'Interfaces', such as a JointInterface, CartesianInterface, ForceInterface, etc.